nautobot/core/views/generic.py

```diff
@@ -252,9 +255,25 @@ def get(self, request):
                 )
                 self.queryset = self.queryset.none()
 
+            def check_for_display(field_name, values):
+                values = values if isinstance(values, (list, tuple)) else [values]
+
+                if field_name not in filterset_filters.keys():
+                    return [field_name, values]
+
+                label = filterset_filters[field_name].label
+                if len(values) == 0 or not is_uuid(values[0]):
+                    return [label, values]
+                else:
+                    try:
+                        filtered_results = filterset_filters[field_name].queryset.filter(pk__in=values)
+                        new_values = [result.display for result in filtered_results]
+                        return [label, new_values]
+                    except (FieldError, AttributeError):
```

Comment:
Just curious, which statement could cause a FieldError?

Reply:
`result.display`, just in case the object doesn't have a `display` property

Reply:
Ahhh i thought that was `AttributeError`